test/serializers/attributes_test.rb

```diff
@@ -23,6 +23,11 @@ def test_attributes_with_fields_option
           @profile_serializer.attributes(fields: [:name]))
       end
 
+      def test_attributes_with_fields_method
```

Comment:
Added test cases to cover no overridden fields and combination of fields method and option.